lib/pbench/server/api/resources/users_api.py

```diff
@@ -299,19 +300,43 @@ def __init__(self, logger, auth):
         self.logger = logger
         self.auth = auth
 
-    def get_valid_target_user(self, target_username: str) -> User:
+    def get_valid_target_user(
+        self, target_username: str, rest_call: str
+    ) -> Tuple["User", HTTPStatus, str]:
```

Comment:
Anonymous tuple returns are really hard to follow; a named tuple would be better. That essentially provides a custom object with attributes that can be read by name... but can also be destructured just like a normal tuple. Seems to provide a lot of benefits in use and self-documentation with very little added complexity.

Reply:
I was envisioning a `Tuple["User", Tuple[HTTPStatus, str]]`, but a [named tuple](https://docs.python.org/3/library/collections.html#collections.namedtuple) would be even better.